test/over_react/component/error_boundary_test.dart

```diff
@@ -47,7 +46,9 @@ void main() {
         document.body.append(mountNode);
         var jacketOfFlawedComponentWithNoErrorBoundary = mount(Flawed()(), mountNode: mountNode);
         expect(mountNode.children, isNotEmpty, reason: 'test setup sanity check');
-        jacketOfFlawedComponentWithNoErrorBoundary.getNode().click();
+        final buttonThatShouldThrowAnErrorWhenClicked = queryByTestId(jacketOfFlawedComponentWithNoErrorBoundary.getInstance(), 'flawedComponent_flawedButton');
+        expect(buttonThatShouldThrowAnErrorWhenClicked, isNotNull, reason: 'test setup sanity check');
+        buttonThatShouldThrowAnErrorWhenClicked.click();
```

Comment:
#nit is there a best practice about when to add space between lines? This is definitely readable without extra lines, but it's also a time where I would personally add a couple to group things and make it a little easier on the eyes.

Reply:
We don't have any hard and fast rules unfortunately. I usually try to group things - but I'll take another pass at this.

Reply:
Updated in https://github.com/Workiva/over_react/pull/350/commits/320c15f4d808bc07d58455e5c7575082410cf5c5

Reply:
Looks great, thank you for addressing that!